README.md

```diff
@@ -17,9 +17,7 @@ As new standards and APIs become available `<model-viewer>` will be improved
 to take advantage of them. If possible, fallbacks and polyfills will be
 supported to provide a seamless development experience.
 
-See: [API](#api), [Demo](https://model-viewer.glitch.me),
-[Examples](https://googlewebcomponents.github.io/model-viewer/index.html),
-[Kanban](https://github.com/GoogleWebComponents/model-viewer/projects/1) (for work prioritization and progress)
+[Remixable demo](https://model-viewer.glitch.me) • [Documentation](https://googlewebcomponents.github.io/model-viewer/index.html) • [Kanban](https://github.com/GoogleWebComponents/model-viewer/projects/1) • [Fidelity test results](https://googlewebcomponents.github.io/model-viewer/test/fidelity/results-viewer.html)
```

Comment:
can we say "Demo" instead of "Remixable demo"?

Reply:
Also, "Fidelity test results" should probably be "Quality Tests"

Reply:
👍 Will fix